poutyne/framework/experiment.py

```diff
@@ -239,7 +239,9 @@ def train(self,
               epochs=1000,
               steps_per_epoch=None,
               validation_steps=None,
-              seed=42):
+              seed=42,
+              batches_per_step=1):
```

Comment:
Whe have a default value too for the constructor. We should only define one default.

Reply:
I think, the model should have the default value not the iterator.

Reply:
Not sure what you mean here: default value has been set to 1 for `batches_per_step` because the default use-case of the whole library is not what this particular PR aims to solve.

Reply:
I think this is fine. However, maybe put the `batches_per_step` argument before the `seed` argument since the `seed` argument is less important and fits more with the previous arguments.

Reply:
And do the same for every other places. I think we should put it after the `validation_steps` argument at every places.

Reply:
Fixed by fecbe56534bf75d572f69f4c09a537505c4e8041